modules/polaris/src/test/ts/atomic/api/RegexesTest.ts

```diff
@@ -230,4 +231,36 @@ UnitTest.test('RegexesTest', () => {
       assert.fail('expected ' + v + ' but did not match "' + k + '"');
     }
   });
+  const onlyWithPathLinks = { // Ignore trailing comma and period in URL path
```

Comment:
Can you please put a newline above this, just to break up the test cases please?